extkeys/hdkey.go

```diff
@@ -95,6 +103,22 @@ var (
 
 	// PublicKeyVersion is version for public key
 	PublicKeyVersion, _ = hex.DecodeString("0488B21E")
+
+	// EthBIP44ParentPath is BIP44 keys parent's derivation path
+	EthBIP44ParentPath = []uint32{
+		HardenedKeyStart + 44,          // purpose
+		HardenedKeyStart + CoinTypeETH, // cointype set to ETH
+		HardenedKeyStart + 0,           // account
+		0,                              // 0 - public, 1 - private
+	}
+
+	// EthEIP1581ChatParentPath is EIP-1581 chat keys parent's derivation path
+	EthEIP1581ChatParentPath = []uint32{
+		HardenedKeyStart + 43,          // purpose
+		HardenedKeyStart + CoinTypeETH, // cointype set to ETH
+		HardenedKeyStart + 1581,        // EIP-1581 subpurpose
+		HardenedKeyStart + 0,           // key_type (chat)
```

Comment:
> I wonder why we don't have HardenedKeyStart + 0 in EthBIP44ParentPath but here it is so.

As defined in the [BIP44 specs](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#path-levels), the 0 in the EthBIP44ParentPath is the `change` which is not hardened, so it starts with 0.

> Also, in this case, 0 means key_type = chat. If it happens that we need one more type, we would use HardenedKeyStart + 1 and so on?

Yes we will have a different index for each key_type, in the same way we have coin types for BIP44 keys. For example `60` is ETH etc..
Instead of using `0` I'm adding a constant